webpack.config.js

```diff
@@ -66,8 +70,12 @@ module.exports = {
   entry: './src/application.js',
   output: {
     path: path.resolve(__dirname, './dist'),
-    filename: 'application.js',
-    sourceMapFilename: 'application.js.map',
+    filename: process.env.NODE_ENV === 'production'
```

Comment:
I think it’s worth declaring a local `const isProduction` since we’re asking this question several times.